satellite/satellitedb/overlaycache.go

```diff
@@ -116,7 +116,7 @@ func (cache *overlaycache) Get(ctx context.Context, id storj.NodeID) (*pb.Node,
 		dbx.OverlayCacheNode_NodeId(id.Bytes()),
 	)
 	if err == sql.ErrNoRows {
-		return nil, overlay.ErrNodeNotFound
+		return nil, errs.Combine(overlay.ErrNodeNotFound, errs.New("couldn't find nodeID: %s", id.String()))
```

Comment:
`errs.Combine` is for combining several different errors into one. Here we actually want to format a single error. So we should change `overlay.ErrNodeNotFound` to be an error class instead of an error. Then instead of using `errs.Combine`, we would call `overlay.ErrNodeNotFound.New(id.String())`.

See `overlay.ErrNotEnoughNodes` for reference.

Reply:
ah okay got it. fixed now!